Fixtures/Miscellaneous/TestSingleFailureXCTest/Sources/TestFailures/TestFailures.swift

```diff
@@ -0,0 +1,2 @@
+// The Swift Programming Language
```

Comment:
You can create a fixture package with just a test target, so you wouldn't need files like this one.

Reply:
I just ran `swift package init` and committed whatever what there. I'll remove the unnecessary files.